sq/src/sq-usage.rs

```diff
@@ -399,6 +400,9 @@
 //!     -h, --help
 //!             Prints help information
 //!
+//!         --sdkms-exportable
+//!             (DANGER) Configure the secret SDKMS key to be exportable in the
+//!             future
```

Comment:
maybe we can remove the in the future, it's a bit confusing. makes me think that we don't support this feature in sdkms but will support it later.

do we mean private key by secret key here? I think all keys are secret keys:)

Reply:
Good points!

- [ ] Change confusing phrase